src/frontend/csrs-portal/src/app/components/questionnaire/questionnaire.component.ts

```diff
@@ -0,0 +1,134 @@
+import { AfterViewInit, Component } from '@angular/core';
+
+@Component({
+  selector: 'app-questionnaire',
+  templateUrl: './questionnaire.component.html',
+  styleUrls: ['./questionnaire.component.scss']
+})
+export class QuestionnaireComponent implements AfterViewInit {
+
+  public bceIdLink: string;
+  //Question 1
+  a1Yes: boolean = false;
```

Comment:
we already have yes or no pipe in shared folder, can you make use of it

Reply:
How can I use yes or no pipe for questionnaire? There are some questions with yes, no  and I don't know responses.